lib/workers/repository/update/branch/index.ts

```diff
@@ -570,9 +570,23 @@ export async function processBranch(
       await scm.checkoutBranch(config.baseBranch);
       updatesVerified = true;
     }
-    // istanbul ignore if
-    if (branchPr && platform.refreshPr) {
-      await platform.refreshPr(branchPr.number);
+
+    if (branchPr) {
+      const platformOptions = getPlatformPrOptions(config);
+      if (
+        platformOptions.usePlatformAutomerge &&
+        config.autoMergeAllowed &&
```

Comment:
Are you sure that this is visible at the worker level it's used in here? I thought it might only be visible in platform/github/index.ts

Reply:
I'm not terribly sure; I'll admit I've been leaning on Typescript a lot for this change. I'll try to figure that out!

If not, I think checking only `usePlatformAutomerge` would still broadly accomplish the goal of "avoid an extra API call when automerge isn't enabled"?

Reply:
Following the flow from `renovateRepository` > `initRepo` > `updateRepo` > `update` > `writeUpdates` > `processBranch`, I wasn't quite able to understand if `config.autoMergeAllowed` would be visible here or not. Still, it does seem like a check that shouldn't happen at this level, so I removed it in my latest push